src/TreeHouse/BaseApiBundle/Behat/ApiContext.php

```diff
@@ -321,12 +321,21 @@ public function theResultKeyShouldEqualNode($key, $node)
     }
 
     /**
-     * @Then the api error should contain ":text"
+     * @Then the api/API error should contain :text
```

Comment:
Sorry I disagree. 

Following your reasoning; if we are going to cater different variants just because some people prefer it, then why not add every possible word? 

For example: `Then (the) api/API error/fault should/will contain/have :text`

It's added obfuscation that's just not needed; once people accept one way of writing something it's better to use that instead of having to support different variants. This is especially important for public software like this bundle.